src/Main.java

```diff
@@ -1,5 +1,67 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        byte a = 127;
+        short b = -32768;
+        int c = 2147483647;
+        long d = -9223372036854775808L;
+        float e = 1.1234567f;
+        double f = 1.1234567891234567;
+        System.out.println("Значение переменной «a» с типом byte равно " + a);
+        System.out.println("Значение переменной «b» с типом short равно " + b);
+        System.out.println("Значение переменной «c» с типом int равно " + c);
+        System.out.println("Значение переменной «d» с типом long равно " + d);
+        System.out.println("Значение переменной «e» с типом float равно " + e);
+        System.out.println("Значение переменной «f» с типом double равно " + f);
+
+        float q = 27.12f;
+        long w = 987678965549L;
+        float o = 2.786f;
+        short r = 569;
+        short t = -159;
+        short y = 27897;
+        byte u = 67;
+        System.out.println(q);
+        System.out.println(w);
+        System.out.println(o);
+        System.out.println(r);
+        System.out.println(t);
+        System.out.println(y);
+        System.out.println(u);
+
+        byte studentsOfTheFirstTeacher = 23;
+        byte studentsOfTheSecondTeacher = 27;
+        byte studentsOfTheThirdTeacher = 30;
+        short totalSheets = 480;
+        System.out.println("На каждого ученика рассчитано " + totalSheets / (studentsOfTheFirstTeacher + studentsOfTheSecondTeacher + studentsOfTheThirdTeacher) + " листов бумаги");
+
+        byte equipmentPerformancePerMinute = 8;
```

Comment:
никогда не нужно считать ничего в уме, для этого мы как раз пишем программы. Было дано 16 бутылок за 2 минуты, значит так и надо было записать
```suggestion
        byte equipmentPerformancePerMinute = 16 / 2;
```